public/js/custom-hj.js

```diff
@@ -6,6 +6,12 @@ jQuery(function($) {
     $('.gp-book-online').on('click', function () {
       hj('tagRecording', ['Book Online']);
     });
+    $('.repeat-prescription-online').on('click', function () {
```

Comment:
On `gp-finder` we used `.on('touchstart click')` as there were some instances where click wasn't working on mobile.